### ISSUE TITLE
Softwareupdate Tesla falsche Mitteilung

### ISSUE THREAD

**Issue by Xoffroad** [Urkman/s3xy-watch-for-tesla#9]
Beim Start und Ende eines Softwareupdates des Tesla erscheint jeweils eine Meldung zum Ladeport. Ich konnte nur einen Screenshot von der Meldung beim Ende des Updates erstellen. Beim Start des Updates ist die Meldung ähnlich oder gleich.
![IMG_1389](https://user-images.githubusercontent.com/65235705/229985671-8a0987a3-5f7e-4a9c-bb0a-b0049fe57e9d.jpeg)

**Comment by Urkman** [comment]
Ja, das ist mir auch schon aufgefallen. Keine Ahnung, warum die API dann sagt, das der Tesla "angeschlossen".
Muss ich beim nächsten Laden mal debuggen...
